docs/content/installation/install_compatibility_requirements.md

```diff
@@ -83,6 +83,16 @@ Complete these steps to prepare your environment for installing the CSI (Contain
 
    The instructions and relevant YAML files to enable volume snapshots can be found at: [https://github.com/kubernetes-csi/external-snapshotter#usage](https://github.com/kubernetes-csi/external-snapshotter#usage)
 
+5. (Optional) If planning on using dynamic volume groups, enable support on your orchestration platform cluster and storage system.
+    
+    1. To enable support on your Kubernetes cluster, install the following replication CRDs once per cluster.
+
+        ```
+        PATHING TO BE ADDED
```

Comment:
curl -O https://raw.githubusercontent.com/IBM/csi-volume-group-operator/v0.9.0/config/crd/bases/csi.ibm.com_volumegroupclasses.yaml
kubectl apply -f csi.ibm.com_volumegroupclasses.yaml
curl -O https://raw.githubusercontent.com/IBM/csi-volume-group-operator/v0.9.0/config/crd/bases/csi.ibm.com_volumegroupcontents.yaml
kubectl apply -f csi.ibm.com_volumegroupcontents.yaml
curl -O https://raw.githubusercontent.com/IBM/csi-volume-group-operator/v0.9.0/config/crd/bases/csi.ibm.com_volumegroups.yaml
kubectl apply -f csi.ibm.com_volumegroups.yaml

Reply:
updated.

Reply:
per @ELENAGER  we have to keep as is for now. we can update later as needed - will be tracking separately as part of CSI-5214.
once you've verified the rest of this thread, please close.thanks

Reply:
reviewed on webex